frontend/src/containers/Profile/ProfileDetail/ProfileDetail.test.js

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import { shallow } from "enzyme";
+import { Link } from "react-router-dom";
+
+import ProfileDetail from "./ProfileDetail";
+
+describe("ProfileDetail Test", () => {
+    it("should render without errors", () => {
+        const component = shallow(<ProfileDetail />);
+        let wrapper = component.find(".ProfileDetail");
+        expect(wrapper.length).toBe(1);
+        wrapper = component.find("#Header");
+        expect(wrapper.length).toBe(1);
+        wrapper = component.find("#SideBar");
+        expect(wrapper.length).toBe(1);
+        wrapper = component.find(".itemTabSection");
+        expect(wrapper.length).toBe(1);
+    });
+
+    // it("click edit button and go to edit page", () => {
```

Comment:
Why did you comment this and some other test blocks?

Reply:
My mistake, I left the old tests commentized while I was working on it.